.elasticbeanstalk/config.yml

```diff
@@ -1,7 +1,3 @@
-branch-defaults:
-  elastic-beanstalk:
-    environment: iit-backend-dev
-    group_suffix: null
 global:
   application_name: iit-backend
   branch: null
```

Comment:
For my understanding, could you explain why the branch is set to `null`? Why not`main` or `master`?

Reply:
This config was just generated when I ran EB init. I would actually prefer that EB environment doesn't have a default branch associated with it so that we are forced to be more verbose (e.g. need to specify the EB env name we are pushing to when using the CLI)